LanguageFeatures/Enhanced-Enum/grammar_A10_t04.dart

```diff
@@ -25,9 +26,6 @@ enum E1 {
   e3;
 
   static void set values(int value) {}
```

Comment:
I'd think this is still an error, because of the mismatch between the parameter type of this setter and the return type of the corresponding (implicitly induced) getter.

Reply:
I agree. I accounted for that error in the header to the enum itself, a few lines above. The getter `values` is generated, so its location is the location of the overall enum. Do you think changing the location to the type of the argument of the setter would be better?

Reply:
Oh, I'm sorry, I did not notice that this was handled by text that hasn't changed (so it wasn't shown in the github diff view until I pressed a button).

So, I'll close this thread.